packages/website/ts/style/colors.ts

```diff
@@ -11,6 +11,8 @@ const appColors = {
     wrapEtherConfirmationButton: sharedColors.mediumBlue,
     drawerMenuBackground: '#4a4a4a',
     menuItemDefaultSelectedBackground: '#424242',
+    jobsPageBackground: sharedColors.grey50,
+    jobsPageOpenPositionRow: sharedColors.grey100,
```

Comment:
I think these `greyX` colors come from Material UI, any way the greys we have define ourselves work?

Reply:
they are actually defined in `react-shared`